api/src/pcapi/scripts/backoffice_users/add_permissions_to_staging_specific_roles.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from flask import Blueprint
+
+from pcapi import settings
+from pcapi.core.permissions.models import Permission
+from pcapi.core.permissions.models import Permissions
+from pcapi.core.permissions.models import Role
+from pcapi.core.permissions.models import RolePermission
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+blueprint = Blueprint("add_permissions_to_staging_specific_roles", __name__)
+
+
+def _purge_role_permissions(role: Role) -> None:
+    RolePermission.query.filter(RolePermission.roleId == role.id).delete()
+
+
+@blueprint.cli.command("add_permissions_to_staging_specific_roles")
+def add_permissions_to_staging_specific_roles() -> None:
+    """Add permissions to qa and global access roles"""
+
+    if settings.IS_PROD:
+        raise ValueError("This function is not supposed to be run on production")
```

Comment:
un logger avec un return serait plus coherent avec la suite de la fonction
```suggestion
         logger.error("This function is not supposed to be run on production"')
        return
```